api/services/stores/mongodb.js

```diff
@@ -9,5 +9,10 @@ const mongoStore = require('cache-manager-mongodb')
  */
 module.exports = (config) => {
   config.store = mongoStore
-  return cacheManager.caching(config)
+  return new Promise((resolve) => {
+    config.createCollectionCallback = () => {
+      return resolve(mongoCache);
```

Comment:
Please remove the semicolon
/home/travis/build/trailsjs/trailpack-cache/api/services/stores/mongodb.js
14:33  error  Extra semicolon  semi